rpc/client/http/http.go

```diff
@@ -201,7 +201,7 @@ func (b *BatchHTTP) Count() int {
 //-----------------------------------------------------------------------------
 // baseRPCClient
 
-func (c *baseRPCClient) Status() (*ctypes.ResultStatus, error) {
+func (c *baseRPCClient) Status(ctx context.Context) (*ctypes.ResultStatus, error) {
 	result := new(ctypes.ResultStatus)
 	_, err := c.caller.Call("status", map[string]interface{}{}, result)
```

Comment:
I think this needs to be propagated into `Call()`. As it is now, it doesn't look like the context is actually used for anything?